java/typequals/vindex/VIndexAnnotatedTypeFactory.java

```diff
@@ -11,6 +11,7 @@
 import org.checkerframework.javacutil.AnnotationBuilder;
 import org.checkerframework.javacutil.TreeUtils;
 import org.checkerframework.javacutil.TypesUtils;
+import typequals.vindex.qual.VIndexTop;
```

Comment:
Why is this import statement needed?

Reply:
I get an error in line 26 of that class if I remove the import.